src/physics/thermal_conduction.cpp

```diff
@@ -106,7 +119,25 @@ void ThermalConduction::completeSetup()
   if (timestepper_ != serac::TimestepMethod::QuasiStatic) {
     // If dynamic, assemble the mass matrix
     M_form_ = temperature_.createOnSpace<mfem::ParBilinearForm>();
-    M_form_->AddDomainIntegrator(new mfem::MassIntegrator());
+
+    // If both a specific heat capacity and density have been specified, use their product as the mass matrix
+    // coefficient
+    if (cp_ != nullptr && rho_ != nullptr) {
+      auto product_function = [](double a, double b) { return a * b; };
+      mass_coef_ = std::make_unique<TransformedScalarCoefficient>(std::move(rho_), std::move(cp_), product_function);
```

Comment:
Out of curiosity, is this any different than `mfem::ProductCoefficient`? If not, that might be preferable if performance is a concern - `TransformedCoef` is of course more flexible but incurs additional `std::function` overhead.

Reply:
Nope - I simply forgot product coefficient existed!